NEWS

```diff
@@ -11,6 +11,9 @@ Flocker 0.4.1dev1 (2015-05-25)
 - There are new systemd units: ``flocker-zfs-agent`` is now ``flocker-container-agent`` and ``flocker-dataset-agent``. (FLOC-1554)
 - ``flocker-reportstate`` and ``flocker-changestate`` have been removed. (FLOC-1663)
 - Updated to Twisted 15.1. (FLOC-1707)
+- Dataset backend support for OpenStack and AWS. (FLOC-1925, FLOC-1743, FLOC-1745)
```

Comment:
This means that the NEWS are slightly out of sync with the title but we will do a release soon to fix this.